sunkit_instruments/suvi/suvi.py

```diff
@@ -0,0 +1,1122 @@
+"""
+This package provides helper routines for the SUVI (Solar UltraViolet Imager) 
+instrument on the GOES-R series of satellites.
+
+.. note::
+    SUVI data (and data from other GOES instrumentation) can be found 
+    `here <https://data.ngdc.noaa.gov/platforms/solar-space-observing-satellites/goes/>`__
+    for L1b and L2 FITS files and
+    `here <https://noaa-goes16.s3.amazonaws.com/index.html>`__ for L1b netCDF files (publicly 
+    available Amazon S3 bucket, look on page 2 after all the ABI data).
+"""
```

Comment:
Can you move this into `docs/code_ref/suvi.rst`?

Reply:
How do I do that? When I just copy&paste it, the tox command for building the documents fails. It is already also in the ```__init__.py``` in the ```suvi/``` folder, so that it shows up at the beginning of the function list document. Is that not where it is supposed to be?

Reply:
And yes, can you please send me some commands to pull in the new main branch? That would be helpful. I am still trying to come to terms with git...

Reply:
It should only really be in one place, I don't like adding text to the init or the top of python files.
You would need to remove it from both files and move it from 

This makes more sense to me in the rst file.

Reply:
> And yes, can you please send me some commands to pull in the new main branch? That would be helpful. I am still trying to come to terms with git...

You will need to do:

```bash
git remote add upstream git@github.com:sunpy/sunkit-instruments.git
git remote update -p
git pull upstream main
```
the first line you can replace with the https version if you use that instead.

That should be enough.